lib/__tests__/util.js

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
+// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license
+
+/* eslint-env jest */
+
+const { cStyleComment, commonComment, indent } = require("../util");
+
+const comment = `
+lines
+of
+comments
+`.slice(1);
```

Comment:
Open for suggestions. My intention was to make the difference between input / output most comprehensible.